packages/projection/test/operators/Mappers/withUtxo.test.ts

```diff
@@ -57,7 +58,16 @@ export const validTxSource$ = of({
               address:
                 'addr_test1qzrf8t56qhzcp2chrtn7deqhep0dttr3eemhnut6lth3gulj7cuplfarmnq5fyumgl0lklddvau9dhamaexykljzvpyswqt56p',
               value: {
-                assets: new Map(),
+                assets: new Map([
```

Comment:
It would be better to keep this output as-is and add a new one instead. This way we can test that we actually filter the outputs. Right now, not applying `filterProducedUtxoByAssetsPresence()` would probably still pass the test.

Reply:
addressed here: 69d8eadfc8d48433777d2ceae53f1a4bdd624fac